catalogue/webapp/services/catalogue/works.js

```diff
@@ -67,6 +69,28 @@ export async function getWorks({
   }
 }
 
+export async function getWorkTypeAggregations({
+  filters,
+  unfilteredSearchResults,
+  env = 'prod',
+}: any): Promise<CatalogueAggregationBucket[]> {
+  const filterQueryString = Object.keys(removeEmptyProps(filters)).map(key => {
+    const val = filters[key];
+    return key !== 'workType' && `${key}=${val}`;
+  });
+  const url =
+    `${rootUris[env]}/v2/works?include=${workIncludes.join(
+      ','
+    )}&aggregations=workType` +
+    (unfilteredSearchResults ? '' : `&workType=${defaultWorkTypes.join(',')}`) +
```

Comment:
You shouldn't need to do this here - it would be good to go through what is trying to be achieved as the parsers should deal with this as they do in the other request.

Reply:
It's because we strip out the workType from the queryString here https://github.com/wellcometrust/wellcomecollection.org/pull/4864/files#diff-b2e86fdb73cc988e0d673c99635e5d51R79
So that we always get all the workTypes in the aggregation list, not just the selected one, but we then only want the default ones if you haven't got the unfilteredSearchResults toggle on

Reply:
Ah, yes - I think I remember brining this up as an edge case a while back - and so the answer is we would to not apply the filters to these aggregations, but we would like to apply the `query`.

This creates the issue that we can't, as the API currently stands, return the results and aggregations in 1 response... might need to have a think about how that works.

Can you think of how you might expect the API to respond to this in 1?